src/output.rs

```diff
@@ -1,33 +1,39 @@
-use std::env;
-use std::ffi::OsString;
 use std::io::{self, Write};
-use std::path::PathBuf;
-use std::process::{Child, Command, Stdio};
-
-use shell_words;
+#[cfg(feature = "paging")]
+use std::process::Child;
 
 use crate::config::PagingMode;
 use crate::errors::*;
+#[cfg(feature = "paging")]
 use crate::less::retrieve_less_version;
 
 #[derive(Debug)]
 pub enum OutputType {
+    #[cfg(feature = "paging")]
     Pager(Child),
     Stdout(io::Stdout),
 }
 
 impl OutputType {
     pub fn from_mode(mode: PagingMode, pager: Option<&str>) -> Result<Self> {
-        use self::PagingMode::*;
+        let _ = pager;
```

Comment:
Silencing a "unused argument" warning? Could we use `#[allow(unused_variables)]` or similar?

Reply:
Fixed by putting cfg(feature = "paging") on from_mode and exposing OutputType::stdout() for controller.rs to use if building without paging.